Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,46 @@
+.tech {
```

Comment:
set display flex with direction column in this selector

Reply:
yes, that was because you had more divs than needed. the styles should still be applied to the `.tech` by doing this  you won't need all the `display: block` properties in the child elements.